src/sensors/mpu9250sensor.cpp

```diff
@@ -167,15 +166,13 @@ void MPU9250Sensor::getMPUScaled()
 {
     float temp[3];
     int i;
+
+#if defined(_MAHONY_H_) || defined(_MADGWICK_H_)
     int16_t ax,ay,az,gx,gy,gz,mx,my,mz;
```

Comment:
```suggestion
    int16_t ax, ay, az, gx, gy, gz, mx, my, mz;
```

Reply:
okay okay. Does the code in the file need to be consistent? I'll fix mpu9250sensors.cpp file

Reply:
If you copy stuff, you should also fix the style inconsistencies in the code that you are copying.

Reply:
that seems like an implicit rule. I didn't know that. thaks to you.
if i edit or copy the code, i will remember your comment.